### ISSUE TITLE
Update Support/ Compatibility Information on Scratch FAQ Page

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#546]
Re case (removed), the information for what software we support on the FAQ page needs to be updated to reflect the browser info and AIR/ offline editor info.

FAQ Page: https://scratch.mit.edu/info/faq/

I'm not really sure who would want to write the copy for this... maybe @kaschm would know? (If not, feel free to direct me to someone else.) : )

EDIT: removed links to private repositories.

**Comment by jwzimmer** [comment]
Moving to scratchr2 because I think that's where the FAQ Page is...

**Comment by TheGrits** [comment]
The FAQ just hasn't been migrated quite yet: https://github.com/LLK/scratch-www/issues/307

**Comment by TheGrits** [comment]
Looks like @technoboy10 has a PR to migrate it at #542

**Comment by jwzimmer** [comment]
@TheGrits, thank you for pointing that out. : )
When #542 is finished, we can reopen this case and close scratchr2 issue 3686. (Scratch-ops case 129 has the compatibility info.)

**Comment by TheGrits** [comment]
@jwzimmer OK, looks like @technoboy10  merged it and the issue can be reopened.

**Comment by jwzimmer** [comment]
@technoboy10, @TheGrits, Great, thank you! : )

**Comment by kaschm** [comment]
yes! you can send content through me :)

**Comment by jwzimmer** [comment]
@kaschm - Thoughts on content options?

Current Version:
<img width="605" alt="screen shot 2016-07-19 at 5 31 04 am" src="https://cloud.githubusercontent.com/assets/1320456/16945182/15deabf4-4d72-11e6-89b5-9a613e073243.png">

Proposed New Version:
- "The Scratch 2 offline editor (beta version) is now available." --> "The Scratch 2 offline editor is now available"
  - The link should still point to the same page, https://scratch.mit.edu/scratch2download/
- "To run Scratch 2, you need a relatively recent web browser (Chrome 35 or later, Firefox 31 or later, or Internet Explorer 9 or later) with Adobe Flash Player version 10.2 or later installed." --> "To run Scratch 2, you need a relatively recent web browser (one of the latest two versions of Chrome, Firefox, Safari, or Edge, or Internet Explorer 10+) with Adobe Flash Player installed."

I'm not sure how to word this simply but accurately... should I mention that not all of these browsers are supported for each OS?

And for Adobe Flash Player, we require a version released on or after June 16, 2016... well, we don't require it, but Adobe required it, because they fixed a critical security issue and told everyone to upgrade: https://helpx.adobe.com/security/products/flash-player/apsb16-18.html. However, the specific version is different depending on browser/ OS. 

More detailed but less simple new versions:
"To run Scratch 2, you need a relatively recent web browser (Chrome 35 or later, Firefox 31 or later, or Internet Explorer 9 or later) with Adobe Flash Player version 10.2 or later installed." --> 
- "To run Scratch 2, you need a relatively recent web browser (one of the latest two versions of Chrome, Firefox, Safari, or Edge, or Internet Explorer 10+) with Adobe Flash Player 22+ installed."
- "To run Scratch 2, you need a relatively recent web browser (one of the latest two versions of Chrome, Firefox, Safari, or Edge, or Internet Explorer 10+) with Adobe Flash Player installed. The specific version of Adobe Flash Player you need depends on your browser and OS; in general, Scratch should work with the most recent version of Adobe Flash Player."
- "To run Scratch 2, you need a relatively recent web browser (one of the latest two versions of Chrome, Firefox, Safari, or Edge, or Internet Explorer 10+) with the latest Adobe Flash Player installed. Scratch is not supported for every OS, or every OS & browser combination. If you would like more information about support for Scratch, please email help@scratch.mit.edu."
- "To run Scratch 2, you need (1) a relatively recent web browser (one of the latest two versions of Chrome, Firefox, Safari, or Edge, or Internet Explorer 10+); (2) Mac, Linux, or Windows OS; (3) a version of Adobe Flash Player released after June 15, 2016." (or "on or after June 16, 2016"?)
- "To run Scratch 2, you need a relatively recent web browser (one of the latest two versions of Chrome, Firefox, Safari, or Edge, or Internet Explorer 10+) and a version of Adobe Flash Player released after June 15, 2016."
- "To run Scratch 2, you need a version of Adobe Flash Player released after June 15, 2016, and...
  - If you are on Mac OS: one of the latest two versions of Chrome, Safari, or Firefox
  - If you are on Linux: one of the latest two versions of Chrome or Firefox
  - If you are on Windows: one of the latest two versions of Chrome, Safari, Firefox, Edge, or Internet Explorer 10+."

Or maybe we could just include a chart like this one along with the article? (Although then maybe @carljbowman would need to design it/ approve it?)
<img width="708" alt="screen shot 2016-07-19 at 5 51 48 am" src="https://cloud.githubusercontent.com/assets/1320456/16945862/ed40ee3e-4d74-11e6-90c8-a9767fbb93ad.png">

**Comment by kaschm** [comment]
Thanks for thinking about this so carefully, @jwzimmer. Here's what I think we should do:

re: Downloadable Version, new text:

"The Scratch 2 offline editor (beta version) is now available." --> "The Scratch 2 offline editor allows you to create Scratch projects without an internet connection. You can download Scratch 2 from the [website](scratch.mit.edu/scratch2download). You can also use the older version, Scratch 1.4. ... etc" 

re: Compatibility Matrix:

So many good option to choose from. How about a slightly tweaked version:

"To run Scratch 2, you need to be using (1) a Mac, Linux, or Windows computer; (2) a version of Adobe Flash Player released on or after June 15, 2016; (3) a relatively recent web browser: one of the latest two versions of Chrome, Firefox, Safari (not including Linux), Edge (Win only), or Internet Explorer 10+ (Win only)"

Any other edits or thoughts?

**Comment by jwzimmer** [comment]
@mewtaylor & @kaschm (& @cwillisf) I have a quick question about this, re [line 16](https://github.com/LLK/scratch-www/pull/819/files): it says "if your computer doesn't meet these requirements, you can try downloading and installing scratch 1.4...", does that mean that scratch 2.0 online & scratch 2.0 offline have the same technical requirements? So if they can't use scratch 2.0 website, they also can't use scratch 2.0 offline?

I think there's another case with the scratch 2.0 requirements, I'm going to look for it so we can make sure this is accurate...

**Comment by cwillisf** [comment]
Scratch 2.0 online & offline have the same system requirements, though they have slightly different software requirements (Adobe Flash for online, Adobe AIR for offline).

The difference is generally only important on Linux. The TL;DR is that Linux users will have the best experience by running Chrome and using the online editor, though the "Linux / older Mac" version of the offline editor should work too.

Some schools and other organizations restrict or centralize software installation, in which case they need to be explicitly aware of the Flash / AIR requirements if they want to run Scratch.

Bottom line: there aren't many situations where downloading the 2.0 offline editor will help if they don't meet the requirements for the 2.0 online editor. The only one I can think of is if you have AIR installed but not Flash, and you're prevented from installing Flash for policy reasons.
